components/PageHeader.vue

```diff
@@ -67,6 +65,7 @@
       submitSearchForm() {
         const newSearchQuery = { ...this.searchQuery, ...{ query: this.query, page: 1 } };
         this.$router.push(this.localePath({ name: 'search', query: newSearchQuery }));
+        this.$root.$emit('submit:searchForm', this.query);
```

Comment:
I wonder if we should be making the most of using Stores for this kind of functionality?

Reply:
Very good point, and I have done so in b5eb3144398802367de4e8f9dc82610bd9bae4f1.